src/reducers/query.js

```diff
@@ -1,20 +1,28 @@
 import { handleActions } from 'redux-actions';
 import immutable from 'immutability-helper';
 import { LOCATION_CHANGE } from 'connected-react-router';
-import { ActionTypes } from 'constants/index';
+import { ActionTypes, TABLE_DEFAULT_ROWS_PER_PAGE } from 'constants/index';
 import BigQuery from 'modules/bigquery';
 
 export const queryState = {
```

Comment:
I'm wondering if its worth renaming this to `previewState`, since it's using the preview API

Reply:
There's so much tied into this "query" name - I think we can push off that name migration to when we ditch the BQ query aspect altogether. At the very least, I'd want to punt to another ticket/PR.